src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/TestFixedDataFactory.kt

```diff
@@ -15,7 +15,7 @@ fun aDifferentValidCertificateStatus() = Status.DISPATCHED
 
 fun aValidCertificateLanguage() = CertificateLanguage.EN
 
-fun aValidCertificateFormat() = CertificateFormat.STANDARD
+fun aValidCertificateFormat() = SupportingInformationFormat.STANDARD
```

Comment:
Minor, @NathanRussellValtech . 
I think it will be good to change the name of the method to `aValidSupportingInformationFormat`, to be consistent.

Reply:
Good shout, yes, I'll do that 👍